requirements.dev.in

```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
+
+pip-tools
```

Comment:
This reminds me that I think we're inconsistent about including `pip-tools` in our dev requirements. I've never figured out whether that's because we've forgotten, or because there's a good reason not too.

Reply:
Asked on Slack 🙂

Reply:
For reference, the answer to this from @inglesp was: yes, because you can then ensure the correct version of pip-tools.